packages/jupyter-ai/package.json

```diff
@@ -72,11 +72,7 @@
     "@mui/icons-material": "^5.11.0",
```

Comment:
Please add `"@jupyterlab/rendermime": "^3.6.0"` to the dependencies.

Reply:
Added as "@jupyterlab/rendermime": "^3.6.3" to match versions of @jupyterlab/application and @jupyterlab/notebook dependencies